includes/wf_crm_webform_postprocess.inc

```diff
@@ -1960,12 +1960,24 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
     $this->saveCustomData($this->data['contribution'][1], $id, 'Contribution', FALSE);
     // Save soft credits
     if (!empty($contribution['soft'])) {
+      // Get total amount either from contribution object or the total amount after line item calculation
+      $amount = $contribution['total_amount'] != null ? $contribution['total_amount'] : $this->totalContribution;
```

Comment:
This does not yield correct results if there is *also* a contribution amount field - and under some circumstances, e.g. a 5¢ additional contribution, reverts to the behaviour where the soft credit is not recorded.
I think better to just use `$this->totalContribution` regardless?